rust/src/transformations/make_stable_expr/test_helper.rs

```diff
@@ -8,7 +8,10 @@ pub fn get_test_data() -> Fallible<(LazyFrameDomain, LazyFrame)> {
         SeriesDomain::new("cycle_5_alpha", AtomDomain::<String>::default()),
         SeriesDomain::new("const_1f64", AtomDomain::<f64>::default()),
         SeriesDomain::new("chunk_(..10u32)", AtomDomain::<u32>::default()),
-        SeriesDomain::new("cycle_(..100i32)", AtomDomain::<i32>::default()),
+        SeriesDomain::new(
+            "cycle_(..100i32)",
+            OptionDomain::new(AtomDomain::<i32>::default()),
+        ),
```

Comment:
I'm not sure how much we gain from having this shared fixture. In this case, it obviously matters for this PR that it's now an `OptionDomain`... but it's not a characteristic that is obvious in the name  `cycle_(..100i32)`, or is obvious downstream in the test where it's used. Are there other tests that used this same fixture, and now they are testing something slightly different because of this change? Perhaps.

The alternative is to have a little bit more duplication in the tests, but tests will be self contained.

I think test fixtures can be useful, but particularly when they're capturing a big example. Here it seems to be another level of indirection without much benefit.

Reply:
Opened in new issue.